libr/bin/format/mach0/mach064_is_kernelcache.c

```diff
@@ -8,6 +8,11 @@ static bool is_kernelcache_buffer(RBuffer *b) {
 		return false;
 	}
 
+	ut32 flags = r_buf_read_le32_at (b, 24);
+	if (!(flags & 0x200000)) {
```

Comment:
maybe worth to use a named constant instead of hardcoding 0x200000 for readability